data-processing/common/commands/fetch_s2_data.py

```diff
@@ -102,10 +105,20 @@ def process(self, item: ArxivId) -> Iterator[S2Metadata]:
                     authors = []
                     for author_data in reference_data["authors"]:
                         authors.append(Author(author_data["authorId"], author_data["name"]))
+
+                    external_ids = reference_data["externalIds"]
+                    if external_ids:
+                        corpus_id = external_ids.get("CorpusId")
+                        arxiv_id = external_ids.get("ArXiv")
+                        doi = external_ids.get("DOI")
```

Comment:
the shape of data is different in the legacy PublicAPI endpoint vs this new one now, see the screenshot in the main comment.

Reply:
It is entirely possible to get `arxiv_id` and/or `doi` to be `None`, just like before.